fontforge/parsettfatt.c

```diff
@@ -6065,6 +6069,7 @@ return;
 	    LogError( _("JSTF table is too long.\n") );
 	    info->bad_ot = true;
 	    free(soff);
+	    free(loff);
```

Comment:
Worried this might be called before the first `realloc` below, so assign NULL up top.

Reply:
I did it on my last patch